lib/dogma/rules.ex

```diff
@@ -1,25 +1,23 @@
 defmodule Dogma.Rules do
-
-  to_rule_name = fn path ->
-    name = path
-            |> Path.basename(".ex")
-            |> Mix.Utils.camelize
-            |> String.to_atom
-    Module.concat( Dogma.Rules, name )
-  end
-
-  files = Path.wildcard( "lib/dogma/rules/*.ex" )
-  rules = Enum.map( files, to_rule_name )
-
-  # FIXME: This doesn't work. Removing a file doesn't trigger recompile.
-  for file <- files do
-    @external_resource file
-  end
+  alias Dogma.Rules
 
   @doc """
-  Returns a list of all modules that define a Dogma rule.
+  Returns a list of all modules that define a Dogma rule and the congiguration
+  that will be passed to each rule.
   """
   def list do
-    unquote(rules)
+    [
+      {Rules.DebuggerStatement},
+      {Rules.FinalNewline},
+      {Rules.LineLength, max_length: 80},
+      {Rules.LiteralInCondition},
+      {Rules.ModuleName},
+      {Rules.NegatedIfUnless},
+      {Rules.QuotesInString},
+      {Rules.TrailingBlankLines},
+      {Rules.TrailingWhitespace},
+      {Rules.UnlessElse},
+      {Rules.WindowsLineEndings}
```

Comment:
That could work but then it would mean that no-one would be able to write their own rule in a different space. So I couldn't have ```MeadSteve.Dogma.CustomRule.AllVariablesStartWithSteve```. I'm not sure what the elixir conventions with namespacing things like this are.